umap/static/umap/js/umap.js

```diff
@@ -29,7 +29,7 @@ L.Map.mergeOptions({
   name: '',
   description: '',
   displayPopupFooter: false,
-  demoTileInfos: { s: 'a', z: 9, x: 265, y: 181, r: '' },
+  demoTileInfos: { s: 'a', z: 9, x: 265, y: 181, '-y': 181, r: '' },
```

Comment:
I don't get that part 🤔

Reply:
Just pushed a comment. Does it make clearer the change ? :)

Reply:
Yup 👍